docs/presentation/Content/PropertySpecification.md

```diff
@@ -12,7 +12,7 @@ This specification allows overriding some attributes of specific ECProperty or d
 | [`overridesPriority`](#attribute-overridespriority)                                                 | No        | `number`                                                          | `1000`      |
 | [`labelOverride`](#attribute-labeloverride)                                                         | No        | `string`                                                          | `undefined` |
 | [`categoryId`](#attribute-categoryid)                                                               | No        | `string \| CategoryIdentifier`                                    | `undefined` |
-| [`isDisplayed`](#attribute-isdisplayed)                                                             | No        | `boolean`                                                         | `undefined` |
+| [`isDisplayed`](#attribute-isdisplayed)                                                             | No        | `boolean \| string`                                                         | `undefined` |
```

Comment:
Table formatting... I suggest adding `yzhang.markdown-all-in-one` extension to VSCode and it'll take care of table formatting for you.